### ISSUE TITLE
TreeView expandTo and expandPath methods do not set aria- and data-expanded attributes properly.

### ISSUE THREAD

**Issue by martintabakov** [telerik/kendo-ui-core#5483]
### Bug report

The expandTo and expandPath methods of the TreeView do not change the data-expanded and aria-expanded attributes to "true".

### Reproduction of the problem

1. Open [this Dojo](https://dojo.telerik.com/UhevojEx/12), which demonstrates two different TreeViews using the two methods.
2. Inspect the expanded items.

### Current behavior
After the items are expanded, their data-expanded and the aria-expanded attributes are false.

### Expected/desired behavior
After the items are expanded, their data-expanded and the aria-expanded attributes should be set to true.

### Environment

* **Kendo UI version:** 2019.3.1023
* **Browser:** [all]
